src/client/src/ui/spotTile/components/notifications/NotificationContainer.tsx

```diff
@@ -83,6 +83,6 @@ export default class NotificationContainer extends PureComponent<Props> {
         )
       }
     }
-    return () => null
+    return () => null as SFC<void>
```

Comment:
You'll get this anyways: 
`Type 'null' cannot be converted to type 'StatelessComponent<void>'.`

Reply:
Nope it will work and without it TS will take issue with implicit any